src/fheroes2/gui/interface_list.h

```diff
@@ -556,6 +578,11 @@ namespace Interface
         fheroes2::TimedEventValidator _timedButtonPgUp;
         fheroes2::TimedEventValidator _timedButtonPgDn;
 
+        int _lastVisibleId() const
+        {
+            return _topId + std::min( maxItems, _size() ) - 1;
```

Comment:
Hi @modo-lv, what about making this expression a bit more clear. When the list is smaller than the `maxItems` the `_topId` will be equal to `0` (but if for some reason it will be not 0) so we can rewrite it:
```suggestion
            return std::min( _topId + maxItems, _size() ) - 1;
```
So now it will say - we return the minimum position of last showed item (starting from the `_topId`) and the last item in the whole list.

Reply:
Done.